src/test/ui/in-band-lifetimes/mismatched_trait.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0621]: explicit lifetime required in the type of `y`
+  --> $DIR/mismatched_trait.rs:16:9
+   |
+15 |     fn baz(&self, x: &'a u32, y: &u32) -> &'a u32 {
```

Comment:
Love it. Happy the errors still work. Not that there is any reason they wouldn't.